input/blog/2020-11-05-cake-v1.0.0-rc0001-released.md

```diff
@@ -0,0 +1,234 @@
+
+### Why Release Candidate?
+
+We've chosen to release 1.0 as a pre-release to NuGet and Chocolatey, so we and the community easier can test and ensure that our 1.0 release will be is as stable as possible.
+It will also enable us to see if there are any steps you need to take to adapt your scripts for 1.0, and for us to see if there're any unintended breaking changes, test as many addins, modules, and recipes as possible.
+
+### Major new features
+
+* .NET 5 support
+* C# 9 support
+* Implicit bootstrapping
+* Rewritten CLI
+* Frosting incorporated into main repo
+* Improved globbing support
+* Improved documentation
+* Bugfixes, increased test coverage, and integration tests.
+
+<!--excerpt-->
+
+### .NET 5 Support
+
+Cake 1.0 will fully support running natively under .NET 5 both with scripts using the .NET Tool [Cake.Tool](https://www.nuget.org/packages/Cake.Tool/) and as regular .NET Console applications using the [Cake.Frosting](https://www.nuget.org/packages/Cake.Frosting/) library.
+
+### C# 9 support
+
+With .NET 5 and the latest version of Roslyn comes the full power of C# 9 to your build process.
+And if we may say so, records are brilliant for your build state
+
+```csharp
+public record StorageAccount(
+    string Name,
+    string Container,
+    string Key
+);
+
+public record BuildData(
+    string Version,
+    DotNetCoreMSBuildSettings MSBuildSettings,
+    StorageAccount StorageAccount
+)
+{
+    public bool ShouldPublish { get; } = !string.IsNullOrEmpty(StorageAccount.Name)
+                                            && !string.IsNullOrEmpty(StorageAccount.Container)
+                                            && !string.IsNullOrEmpty(StorageAccount.Key);
+}
+
+Setup<BuildData>(context=>{
+    var version = context.Argument("version", "1.0.0");
+
+    return new (
+        version,
+        new DotNetCoreMSBuildSettings()
+                                .WithProperty("Version", version)
+                                .WithProperty("Configuration", context.Argument("configuration", "Release")),
+        new (
+            context.EnvironmentVariable("PUBLISH_STORAGE_ACCOUNT"),
+            context.EnvironmentVariable("PUBLISH_STORAGE_CONTAINER"),
+            context.EnvironmentVariable("PUBLISH_STORAGE_KEY")
+        )
+    );
+});
+
+Task("Build")
+    .Does<BuildData>((context, data) => context.Information("Building {0}", data.Version));
+
+RunTarget("Build");
+```
+
+### Implicit bootstrapping
+
+Boostrapping of Cake modules are now implicit, which means that you no longer will need to execute Cake twice to download and initialize modules, just add the module directive to your script and this will be handled transparently for you.
+
+```c#
+#module nuget:?package=Cake.DotNetTool.Module&version=0.4.0
+```
+
+### Rewritten CLI
+
+We're now utilizing [Spectre.Cli](https://www.nuget.org/packages/Spectre.Cli/) for a more robust, flexible and future prof handling of
```

Comment:
End of sentence seems to be missing

Reply:
3AM no idea what I was thinking... but added something :)